README.md

```diff
@@ -276,13 +276,16 @@ services:
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
-```
-//Put global setup code here
+
+```javascript
+  // jest.config.js
+  globalSetup: './example-application/test/global-setup.js'
+
+  // global-setup.js
+  await dockerCompose.upAll();
```

Comment:
That's mystery - How come we can invoke dockerCompose liks this?

Put more context, require the relevant package, give a bit more context